src/components/Toast/ToastDisplay/index.js

```diff
@@ -77,7 +73,7 @@ class ToastDisplay extends PureComponent /*:: <Props, State> */ {
       <ul
         className={styles['toast-display']}
         onMouseEnter={this._handleMouseEnter}
-        onMouseLeave={this._handleMouseLeave}
+        // onMouseLeave={this._handleMouseLeave}
```

Comment:
Why not supporting this callback anymore?

Reply:
My mistake, I had commented to test some styling and forgot about it